app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    for part in command.split(" "):
+        if part[1] != part[2]:
+            with open(part[1], "r") as file_in, open(part[2], "w") as file_out:
+                file_out.write(file_in)
```

Comment:
The `file_out.write(file_in)` is incorrect. You need to read the content from `file_in` and then write it to `file_out`. Consider using `file_out.write(file_in.read())` instead.